acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature

```diff
@@ -28,18 +28,25 @@ Feature: mod-consortia integration tests
       | 'mod-inventory-storage'     |
       | 'mod-inventory'             |
       | 'mod-circulation-storage'   |
-
+      | 'mod-circulation'           |
+      | 'mod-feesfines'             |
 
     * table adminAdditionalPermissions
       | name                                         |
       | 'orders-storage.module.all'                  |
       | 'finance.module.all'                         |
+      | 'circulation.all'                            |
+      | 'overdue-fines-policies.item.post'           |
+      | 'lost-item-fees-policies.item.post'          |
       | 'acquisitions-units.memberships.item.delete' |
       | 'acquisitions-units.memberships.item.post'   |
       | 'acquisitions-units.units.item.post'         |
 
+    * callonce variablesCentral
+
     # generate names for tenants
-    * def random = callonce randomMillis
+#    * def random = callonce randomMillis
+    * def random = '1719515436303'
```

Comment:
is it ok that random name has constant value, probably need to rename

Reply:
Was leftover, fixed